compiler/plc_driver/src/cli.rs

```diff
@@ -140,7 +140,7 @@ pub struct CompileParameters {
         parse(try_from_str = validate_config),
         // FIXME: For some reason, this does not work at the moment but it really should
         // The binary behaves as expected but the tests fail
```

Comment:
The CI seems to be green, does that mean this is obsolete?

Reply:
yes but i'll check again and delete it